src/test/groovy/edu/ie3/datamodel/utils/validation/ConnectorValidationUtilsTest.groovy

```diff
@@ -0,0 +1,259 @@
+	GridTestData.geoJsonReader.read("{ \"type\": \"LineString\", \"coordinates\": [[7.4116482, 51.4843281], [3.4116482, 10.4843281], [7.4116482, 51.4843281]]}") as LineString,
+	OlmCharacteristicInput.CONSTANT_CHARACTERISTIC
+	)
+
+	def "ConnectorValidationUtils.checkLine() recognizes all potential errors for a line"() {
+		when:
+		ConnectorValidationUtils.check(invalidLine)
+
+		then:
+		Exception ex = thrown()
+		ex.class == expectedException.class
+		ex.message == expectedException.message
+
+		where:
+		invalidLine                                                                                                                                                                                   || expectedException
+		GridTestData.lineFtoG.copy().nodeA(GridTestData.nodeG).build()                                                                                                                                || new InvalidEntityException("LineInput connects the same node, but shouldn't", invalidLine)
+		GridTestData.lineFtoG.copy().nodeA(GridTestData.nodeF.copy().subnet(5).build()).build()                                                                                                       || new InvalidEntityException("LineInput connects different subnets, but shouldn't", invalidLine)
+		GridTestData.lineFtoG.copy().nodeA(GridTestData.nodeF.copy().voltLvl(GermanVoltageLevelUtils.MV_10KV).build()).build()                                                                        || new InvalidEntityException("LineInput connects different voltage levels, but shouldn't", invalidLine)
+		GridTestData.lineFtoG.copy().length(Quantities.getQuantity(0d, METRE)).build()                                                                                                                || new InvalidEntityException("The following quantities have to be positive: 0 km", invalidLine)
+		GridTestData.lineFtoG.copy().nodeA(GridTestData.nodeF.copy().geoPosition(testCoordinate).build()).build()                              														  || new InvalidEntityException("Coordinates of start and end point do not match coordinates of connected nodes", invalidLine)
+		GridTestData.lineFtoG.copy().nodeB(GridTestData.nodeG.copy().geoPosition(testCoordinate).build()).build()                              														  || new InvalidEntityException("Coordinates of start and end point do not match coordinates of connected nodes", invalidLine)
+		invalidLineLengthNotMatchingCoordinateDistances 																																			  || new InvalidEntityException("Line length does not equal calculated distances between points building the line", invalidLine)
+	}
+
+	def "Smoke Test: Correct line type throws no exception"() {
+		given:
+		def lineType = GridTestData.lineTypeInputCtoD
+
+		when:
+		ValidationUtils.check(lineType)
+
+		then:
+		noExceptionThrown()
+	}
+
+	// No tests for "ConnectorValidationUtils.checkLineType recognizes all potential errors for a line type"
```

Comment:
Why?

Reply:
no tests necessary, as only other methods are called there which are tested elsewhere. Added this comment to keep a clear structure in the class and doesn't look like those tests were forgotten

Reply:
This explanation should be included in the comment then in order to avoid @ckittl asking why ;-)